app/code/Magento/AmqpStore/Plugin/Framework/Amqp/Bulk/Exchange.php

```diff
@@ -83,9 +83,6 @@ public function beforeEnqueue(SubjectExchange $subject, $topic, array $envelopes
         $updatedEnvelopes = [];
         foreach ($envelopes as $envelope) {
             $properties = $envelope->getProperties();
-            if (!isset($properties)) {
```

Comment:
$properties is always array so I removed this logic
https://github.com/mbvb1223/magento2/blob/5d2f2e87a6f23db0483a53d445bbf055826c022a/lib/internal/Magento/Framework/MessageQueue/Envelope.php#L29